### ISSUE TITLE
FiraCode Retina is not really monospaced

### ISSUE THREAD

**Issue by dojeda** [tonsky/FiraCode#781]
I found a weird example that shows that the retina version of fira code regular is not really monospaced. I am using version 1.206 on TTF. I found this issue using PyCharm 2019.1 on OSX 10.14.3.

Here is the code excerpt:

```python
@pytest.mark.parametrize('predecessor,theirs,mine,expected', [
    ({}, {}, {}, {}),                          # No change at all, key did not exist
    ({'x': 1}, {'x': 1}, {'x': 1}, {'x': 1}),  # No change at all, key did exist
    ({'x': 1}, {'x': 2}, {'x': 2}, {'x': 2}),  # Same change, key did exist
    ({}, {'x': 1}, {'x': 1}, {'x': 1}),        # Same change, key did not exist
    ({}, {'x': 1}, {}, {'x': 1}),              # Their change, key did not exist
    ({'x': 1}, {'x': 2}, {'x': 1}, {'x': 2}),  # Their change, key did exist
    ({}, {}, {'x': 1}, {'x': 1}),              # Mine change, key did not exist
    ({'x': 1}, {'x': 1}, {'x': 2}, {'x': 2}),  # Mine change, key did exist
    ({}, {'x': 1}, {'x': 2}, None),            # Different change, key did not exist
    ({'x': 1}, {'x': 2}, {'x': 3}, None),      # Different change, key did exist
])
def test_3way_merge(predecessor, theirs, mine, expected):
    if expected is None:
        with pytest.raises(Conflict):
            merge(predecessor, theirs, mine)
    else:
        new = merge(predecessor, theirs, mine)
        assert new == expected
```

This is how it looks on Fira Code Retina Regular 13pt:
![image](https://user-images.githubusercontent.com/795818/59566857-fcddc600-9065-11e9-9be7-98d57df7fd06.png)

This is how it looks on Fira Code Regular 13pt:
![image](https://user-images.githubusercontent.com/795818/59566866-0d8e3c00-9066-11e9-89ad-392f05dac68a.png)

Note how lines 2-11 show a weird alignment.

**Comment by tonsky** [comment]
Can you test the same in another editor? If I remember correctly PyCharm come with an old version of Fira Code

**Comment by dojeda** [comment]
Hi, thanks for the response. 

I have tried on other editors, such as TextEdit, and the spacing is correct. 
As you mentioned, PyCharm does have an old version (1.205), so I imagine this was fixed in 1.206. 
I realize now that PyCharm does not use the fonts that I have installed manually on my system and I cannot change PyCharm's fonts. 

So, in conclusion, this seems like a PyCharm issue.

**Comment by tonsky** [comment]
Yeah strange. If I select “Fira Code“ in Idea it uses the one installed in a system. If I choose any other weight (Fira Code Retina, Fira Code Light, Fira Code Medium etc) it uses the bundled version. Reported here https://youtrack.jetbrains.net/issue/IDEA-216433
